resources/js/Pages/Admin/Roles/CreateRoleForm.vue

```diff
@@ -25,7 +25,7 @@
                 <div class="mt-2 grid grid-cols-1 md:grid-cols-2 gap-4">
                     <div v-for="permission in availablePermissions">
                         <label class="flex items-center">
-                            <input type="checkbox" class="form-checkbox" :value="permission.name" v-model="form.permissions">
+                            <jet-checkbox :value="permission.name" v-model="form.permissions"/>
```

Comment:
```suggestion
                            <x-checkbox :value="permission.name" v-model="form.permissions"/>
```
The convention for `x`-tending existing tags is to prefix with an `x`

Reply:
still a jetstream component